eng/pipelines/runtime-ios-scenarios-perf-jobs.yml

```diff
@@ -0,0 +1,140 @@
+parameters:
+  hybridGlobalization: True
+  runtimeRepoAlias: runtime
+  performanceRepoAlias: self
+  jobParameters: {}
+
+jobs:
+  - template: /eng/pipelines/performance/templates/perf-ios-scenarios-build-jobs.yml@${{ parameters.runtimeRepoAlias }}
+    parameters:
+      hybridGlobalization: ${{ parameters.hybridGlobalization }}
+
+  # run mono iOS scenarios HybridGlobalization
```

Comment:
@matouskozak Is `HybridGlobalization` enabled by default?

Reply:
It is, the parameter shouldn't have any effect. We didn't yet do the clean-up in the dotnet/runtime repo to remove it completely (https://github.com/dotnet/runtime/issues/105130).

Reply:
Since there is an outstanding issue for this, will close this comment so it can be addressed as part of that work as I wouldn't be able to verify these changes myself easily.